src/constants/content-report-ts.ts

```diff
@@ -0,0 +1,18 @@
+export const DMCA = 'dmca'
```

Comment:
This PR LGTM but the file name is throwing me off, why is it `content-report-ts.ts` instead of just `content-report.ts`? 

Do we just need to hold off reviewing/merging this one until the content-report module is converted in `main`?

Reply:
I based this PR on @dhruvkb 's PR from branch`tsify_v`. Then, that PR got scaled down, and the changes I was using were passed to `tsify_vi` branch, so the types this was depending on were lost, so I just added a temporary file. Should have set this PR to draft, actually :) 
I've now extracted the @dhruvkb 's changes that this PR relies on into this PR, so now it's a joint PR that is ready to be reviewed.